README.md

```diff
@@ -1,16 +1,286 @@
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "topics": "SAMPLE_TOPIC, ANOTHER_TOPIC, YET_ANOTHER_TOPIC",
+    ...
+  }
+}
+```
+
+#### Basic Configuration with DLQ
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "errors.tolerance": "all",
+    "errors.deadletterqueue.topic.name": "SAMPLE_DLQ_TOPIC",
+    "errors.deadletterqueue.context.headers.enable": "true",
+  }
+}
+```
+
+#### Basic Configuration with DLQ and Multiple Topics
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "topics": "SAMPLE_TOPIC, ANOTHER_TOPIC, YET_ANOTHER_TOPIC",
+    "errors.tolerance": "all",
+    "errors.deadletterqueue.topic.name": "SAMPLE_DLQ_TOPIC",
+    "errors.deadletterqueue.context.headers.enable": "true",
+  }
+}
+```
+
+#### Avro Schema Support
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "value.converter": "io.confluent.connect.avro.AvroConverter",
+    "value.converter.schema.registry.url": "http://localhost:8081",
+    "value.converter.schemas.enable": "true",
+  }
+}
+```
+
+#### JSON Schema Support
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "value.converter.schemas.enable": "true",
+  }
+}
+```
+
+
+#### Exactly-Once Configuration
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    "tasks.max": "1",
+    "database": "default",
+    "errors.retry.timeout": "60",
+    "exactlyOnce": "true",
+    "hostname": "localhost",
+    "password": "[INSERT_PASSWORD_HERE]",
+    "port": "8443",
+    "ssl": "true",
+    "topics": "SAMPLE_TOPIC",
+    "username": "default",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "value.converter.schemas.enable": "false",
+    "clickhouseSettings": ""
+  }
+}
+```
+
+#### Exactly-Once Configuration with DLQ
+```json
+{
+  "name": "clickhouse-connect",
+  "config": {
+    "connector.class": "com.clickhouse.kafka.connect.ClickHouseSinkConnector",
+    ...
+    "errors.tolerance": "all",
+    "errors.deadletterqueue.topic.name": "SAMPLE_DLQ_TOPIC",
+    "errors.deadletterqueue.context.headers.enable": "true",
+  }
+}
+```
+
+### Installing the Connector
+Running an instance of Kafka Connect is a prerequisite for installing the connector - see the [Kafka Connect documentation](https://docs.confluent.io/platform/current/connect/index.html) - but a simple way for development work is using docker compose.
```

Comment:
Connector Framework is part of OSS Apache Kafka. Maybe we should link https://kafka.apache.org/documentation.html#connect as well?